src/app/core/models/input.model.ts

```diff
@@ -1,4 +1,4 @@
 export interface InputData {
-  provider: string,
-  id: string | null
+  provider: string;
```

Comment:
ale to czekaj, bo pytalem chyba o to
tu nie mozemy wrzucic VideoServices? czy ma byc string?